truffle-config.js

```diff
@@ -0,0 +1,4 @@
+const path = require("path");
+const wkdir = path.dirname(require.resolve("@uma/core/package.json"));
+
+module.exports = require("@uma/common").getTruffleConfig(wkdir);
```

Comment:
are you sure we need this if we are using the `yarn link` method to link to core?